config/authorization/config.ex

```diff
@@ -154,6 +170,19 @@ defmodule Acl.UserGroups.Config do
                       ]
                     } },
                    ] },
+      %GroupSpec{
+      name: "o-vendor-api-r",
+      useage: [:read],
+      access: access_for_vendor_api(),
+      graphs: [
+        %GraphSpec{
+          graph: "http://mu.semte.ch/graphs/vendors/",
+          constraint: %ResourceConstraint{
+            resource_types: [
+              "http://rdf.myexperiment.org/ontologies/base/Submission",
+              "http://mu.semte.ch/vocabularies/ext/SubmissionDocument",
+              "http://lblod.data.gift/vocabularies/automatische-melding/FormData",
```

Comment:
Are these the only resources we need vendors to have access to?

Reply:
Don't we also need RemoteDataObjects and FileDataObjects? This would be for the attachments. So:

* http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#RemoteDataObject
* http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#FileDataObject

Reply:
And do we need to export the SubmissionDocument? I tend to forget the model quickly, but isn't the SubmissionDocument the data the vendor submitted themselves in the first place? Is there anything they would want to know from that subject?

I'm asking this because this class is not in the epic description [DL-3947].

Reply:
You're right; these need to be added as well. If I check the [Loket config](https://github.com/lblod/app-digitaal-loket/blob/master/config/authorization/config.ex#L510-L526), I can see the missing resources you mentioned. `RemoteDataObject` is not there however (only `FileDataObject`), so I will confirm locally with a backup to see if this resource is used in submissions.

Reply:
After querying the database, it seems submissions do not have any `RemoteDataObject` resources (only `FileDataObject` ones); I only added the `FileDataObject` resource for now.

Reply:
I took one Submission as an example, and it has some `generated` FormData that `hasPart` many subjects that are both a `FileDataObject` and a `RemoteDataObject`. I suppose that exposing one type in mu-auth allows the object to be fully exposed?

Reply:
It seems that every attachment is always both: https://github.com/lblod/import-submission-service/blob/fcd4ffbdfc530a4be8ca944a2062ea8339bb86ea/lib/attachment-helpers.js#L26C11-L26C88

Reply:
> I took one Submission as an example, and it has some generated FormData that hasPart many subjects that are both a FileDataObject and a RemoteDataObject. I suppose that exposing one type in mu-auth allows the object to be fully exposed?

I remember a discussion with Felix where he mentioned `mu-auth` does not restrict itself to the resources we provide in the configuration. In other words, `mu-auth` is restricted to the graph we provide, but not to the resources we expose in that graph, which can explain what you pointed out (`exposing one type in mu-auth allows the object to be fully exposed`).

Reply:
> It seems that every attachment is always both: https://github.com/lblod/import-submission-service/blob/fcd4ffbdfc530a4be8ca944a2062ea8339bb86ea/lib/attachment-helpers.js#L26C11-L26C88

I'll add `RemoteDataObject` to the configuration to play it defensively.